### DISCUSSION TITLE
Seaborn Facet Grid not rendering as a plot

### DISCUSSION THREAD

**Discussion by toobamukhtar** [datapane/datapane#114] [Help] [answered]
I am creating a facet grid using seaborn and inserting it in dp.Plot(). However, data-pane gives me the following error: "can't embed object as a plot". It embeds simple plots from seaborn but Facet grid is not embedding. Does anyone know a workaround for this?

Thanks!
![error1](https://user-images.githubusercontent.com/29259233/120217191-73732e80-c251-11eb-8de4-0612a32664f1.PNG)

**Comment by johnmicahreid** [comment, marked as answer]
Hi toobamukhtar, could you share the code you used to create the plot?

**Reply by toobamukhtar** [reply]
![plot_a](https://user-images.githubusercontent.com/29259233/120849058-046a4280-c58f-11eb-808c-c272642b9805.PNG)
![plot_b](https://user-images.githubusercontent.com/29259233/120849075-092ef680-c58f-11eb-9003-d0d7c0621181.PNG)

The first image shows the facet grid created using seaborn. It renders correctly in jupyter notebook however when I input it into 
dp.Plot(), it gives an error. Any suggestions on how to display the seaborn Facet plot in datapne report?

**Reply by mands** [reply]
Thanks for the feedback @toobamukhtar - yes it looks like FacetGrid directly is not currently supported. We'll add support for it shortly, however in the meantime you can extract the matplotlib figure from the FacetGrid and pass that to dp.Plot, e.g.

```python
figure = g.get_figure()  # you may need to use `g.fig` depending on your version of seaborn
dp.Plot(figure, caption = "Facet Plot")
```

**Reply by toobamukhtar** [reply]
Thank you!
